### PR TITLE
added unlock section

### PR DESCRIPTION
**Pull request by hoonsubin**
bandicam 2020-03-22 23-07-31-595

added image

### CODE REVIEW

File: PlasmNetwork/Lockdrop.md [hunk 1]
```diff
@@ -120,4 +120,41 @@ The affiliate program mainly has these three rules:
...
+Sending a transaction to the lock (i.e. unlocking the tokens) can be done by anyone given that they have enough funds to pay for the transaction fee.
+But the the locked tokens will only return to the original locker rather than the address that sent the transaction.
+So it is possible to allow another address to unlock the locked token on behalf of the original locker, but they cannot claim the tokens for themselves.
+One minor inconvenience that this contract has is that once the lock duration is over, the contract will act as a normal contract without any transaction rejections, meaning even if the token was returned to the original locker, anyone can still send a transaction to the contract without any errors.
+Once the token is claimed it will not be able to return any more tokens, but it makes it hard for the original locker to check if the tokens were unlocked or not without comparing their post-unlock wallet balance, effectively giving a potential issue of wasting transaction fee for attempting to claim the locked tokens that was already unlocked.
+
+![UnlockFormUI](../img/unlock-ui-from.jpg)
```
Comment (satellitex) [inline PlasmNetwork/Lockdrop.md]: When you use images, you can use uploading on an issue.
https://help.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests

**Comment by satellitex** [comment]
The Unlock topic should is described in another section.
This is because the timing of Locking and Unlocking is different.

**Comment by hoonsubin** [comment]
@satellitex do you have any recommendations on where it should be moved to?

File: PlasmNetwork/Lockdrop.md (hunk 1 above)
Comment (hoonsubin) [reply PlasmNetwork/Lockdrop.md]: added images 
bandicam 2020-03-22 23-07-31-595

**Comment by satellitex** [comment]
@hoonsubin You can use 
`unlock image` on your markdown.

**Comment by hoonsubin** [comment]
> @hoonsubin You can use
> `unlock image` on your markdown.

oh! that is what you meant, sorry I misunderstood.

File: SUMMARY.md [hunk 2]
```diff
@@ -24,3 +24,4 @@
     * [Design Overview](PlasmTestnet/DesignOverview.md)
     * [Validator Guide](PlasmTestnet/ValidatorGuide.md)
     * [Lockdrop](PlasmTestnet/Lockdrop.md)
+
```
Comment (satellitex) [inline SUMMARY.md]: I think the below is better.
```
* [Lockdrop Guide](Trying/Lockdrop.md)
    * [Join the Lockdrop Guide](Trying/Lockdrop.md)
    * [Unlock Guide](Trying/[this PR contents])
```

**Comment by satellitex** [comment]
LGTM

**Pull request merged**